README.md

```diff
@@ -288,9 +288,10 @@ In addition to this, if you want to debug your Blazor pilet using for example Vi
 
 There are some special files that you can add in your project (adjacent to the *.csproj* file):
 
-- *setup.tsx*
-- *package-overwrites.json*
-- *js-imports.json*
+- *.piralconfig/setup.tsx*
```

Comment:
This is breaking - instead of doing that I'd say let's introduce an optional config folder in the csproj which (by default) is "." (i.e., if its not given it resolves to the current folder). This way you can set it to ".piralconfig" (or whatever) and have the files served from there.

Note that files like `js-imports.json` can already be re-defined and remapped. The new setting would therefore only impact the base path.

Reply:
yes thats a good idea. i've added that